tests/Responses/ZendResponseFactoryTest.php

```diff
@@ -27,7 +28,10 @@ public function testCreate()
         $this->assertInstanceOf('Zend\Diactoros\Response', $response);
         $this->assertEquals('image/jpeg', $response->getHeaderLine('Content-Type'));
         $this->assertEquals('5175', $response->getHeaderLine('Content-Length'));
-        $this->assertContains(gmdate('D, d M Y H:i', strtotime('+1 years')), $response->getHeaderLine('Expires'));
```

Comment:
Curious why you changed this from `gmdate` to `date_create`?

Reply:
It's been too long, I can't remember. Should have specified that better. But the tests are working wothout the change, so I'm going to revert that.